tests/rptest/wasm/wasm_test.py

```diff
@@ -74,6 +75,20 @@ def _build_script(self, script):
         self._rpk_tool.wasm_deploy(
             script.get_artifact(self._build_tool.work_dir), "ducktape")
 
+    def restart_wasm_engine(self, node):
+        self.logger.info(
+            f"Begin manually triggered restart of wasm engine on node {node}")
+        node.account.kill_process("bin/node", clean_shutdown=False)
+        # TODO: Write a test where this timeout is 1 second, this is within
+        # the heartbeat window, currently this poses an issue
+        time.sleep(3)
```

Comment:
can you replace this with a test for the property you are trying to achieve with the delay?

Reply:
@graphcareful please leave a link to a ticket describing the issue and other that this pr looks great!

Reply:
Fixed comment and created issue https://github.com/vectorizedio/redpanda/issues/1140